Remotion/Web/Core/Res/HTML/ExecutionEngine.js

```diff
@@ -59,11 +66,21 @@ function WxePage_Context(
 
   var _isCacheDetectionEnabled = isCacheDetectionEnabled;
 
+  var _wxePostBackSequenceNumberFieldID = wxePostBackSequenceNumberFieldID;
+  var _dmaWxePostBackSequenceNumberFieldID = dmaWxePostBackSequenceNumberFieldID;
+
   // Handles the page loading event.
   this.OnLoading = function(hasSubmitted, isCached)
   {
     ArgumentUtility.CheckNotNullAndTypeIsBoolean('hasSubmitted', hasSubmitted);
     ArgumentUtility.CheckNotNullAndTypeIsBoolean('isCached', isCached);
+
+    if (_dmaWxePostBackSequenceNumberFieldID != null)
+    {
+      let dmaWxePostBackSequenceNumberField = document.getElementById(_dmaWxePostBackSequenceNumberFieldID);

```

Comment:
here it's let, above it's 'var'. pelase use var for now if the rest of the method also uses var and there's not benefit of let.